flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/PartitionTestUtils.java

```diff
@@ -40,35 +38,38 @@ public static ResultPartition createPartition(ResultPartitionType type) {
 				false);
 	}
 
-	public static ResultPartition createPartition(ResultPartitionType type, int numChannels) {
-		return createPartition(new NoOpResultPartitionConsumableNotifier(), type, numChannels, false);
-	}
-
 	public static ResultPartition createPartition(
 			ResultPartitionConsumableNotifier notifier,
 			ResultPartitionType type,
 			boolean sendScheduleOrUpdateConsumersMessage) {
 
-		return createPartition(notifier, type, 1, sendScheduleOrUpdateConsumersMessage);
+		return createPartition(null, notifier, type, 1, sendScheduleOrUpdateConsumersMessage);
 	}
 
 	public static ResultPartition createPartition(
```

Comment:
Since we already have the `ResultPartitionBuilder`, this `PartitionTestUtils` seems not very necessary because it seems trouble to maintain the different units of parameters.

Reply:
Well, certain short-cuts are still shorter if used in many places.. Not sure, I'll reduce it a bit more but atm I do not see it strictly being necessary.